install_config/web_console_customization.adoc

```diff
@@ -688,6 +688,51 @@ endif::[]
 ----
 
 endif::[]
+[[configuring-quota-notification-messages]]
+== Configuring Quota Notification Messages
+
+Whenever a user reaches or exceedes a quota a quota notification is put into the notificaiton drawer.
+A custom quota notification message, per
+xref:../dev_guide/compute_resources.adoc#dev-managed-by-quota[quota resource type], can be added to the notificaiton.  For example:
+"Your project is over quota.  It is using 200% of 2 cores CPU (Limit). Upgrade to <a href='http://www.openshift.com'>
+OpenShift Pro</a> if you need additional resources.".  The "Upgrade to..." part of the notification is the custom
+message and may contain HTML/links to additional resources.
```

Comment:
HTML or links

Reply:
Hmm...it's not really an 'or' situation.  I'll update to "...may contain HTML such as links to additional resources".